gradle.properties

```diff
@@ -1,3 +1,3 @@
 slf4jVersion=1.7.12
 groovyVersion=2.4.7
-ratpackVersion=1.4.1
+ratpackVersion=1.4.4
```

Comment:
I don't think we want to bump that, ideally we stay on the lowest version possible to not require consumers to bump the version.